tests/integration/ef_tests/utils.py

```diff
@@ -0,0 +1,62 @@
+import json
+import os
+
+from tests.utils.uint256 import uint256_to_int
+
+# Root of the GeneralStateTest in BlockchainTest format
+EF_GENERAL_STATE_TEST_ROOT_PATH = (
+    "./tests/integration/ef_tests/test_data/BlockchainTests/GeneralStateTests/"
+)
+
+DEFAULT_NETWORK = "Shanghai"
+
+
+def display_storage(uint256_tuple):
```

Comment:
where is this used?

Reply:
https://github.com/jobez/kakarot/blob/feat/add_direct_keyword_for_faster_ef_test_loading/tests/integration/ef_tests/runner.py#L122

value prop here is it lets you see the data in terms of the ef test format and also attempts to mirror how the rust implementation does it

Reply:
remove this plz as per comment